client/index.html

```diff
@@ -7,8 +7,6 @@
     <meta name="theme-color" content="#232835" />
     <meta property="og:image" content="/api/opengraph/image" />
     <meta property="og:image:type" content="image/png" />
-    <meta property="og:image:width" content="402" />
-    <meta property="og:image:height" content="202" />
```

Comment:
I accidentally left the dimensions in for the original height of the image in. 

Since the image generated by satori has different dimensions than the project banner, I figured it was better to remove the meta tags entirely as they aren't strictly needed.